util/chunk/chunk.go

```diff
@@ -525,10 +511,27 @@ func (r Row) GetBytes(colIdx int) []byte {
 }
 
 // GetTime returns the Time value with the colIdx.
+// TODO: use Time structure directly.
 func (r Row) GetTime(colIdx int) types.Time {
 	col := r.c.columns[colIdx]
-	t, _ := col.ifaces[r.idx].(types.Time)
-	return t
+	return readTime(col.data[r.idx*16:])
+}
+
+func readTime(buf []byte) types.Time {
```

Comment:
Is it more reasonable  to make a `Marshal/Unmarshal` for types.Time?
If one day we change types.Time struct but forget here, it may cause some trouble.